src/main/java/seedu/address/logic/parser/TeamCommandsParser.java

```diff
@@ -56,17 +60,21 @@ public static Command parse(String arguments) throws ParseException {
         Index parsedIndex;
 
         switch (subCommand) {
-        case AddTeamCommand.COMMAND_WORD:
-            return new AddTeamCommandParser().parse(args);
-        case ListTeamCommand.COMMAND_WORD:
-            parsedIndex = ParserUtil.parseIndex(index);
-            return new ListTeamCommand(parsedIndex);
-        case DeleteTeamCommand.COMMAND_WORD:
-            parsedIndex = ParserUtil.parseIndex(index);
-            return new DeleteTeamCommand(parsedIndex);
-        default:
-            logger.finer("This user input caused a ParseException: " + arguments);
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, MESSAGE_USAGE));
+            case AddTeamCommand.COMMAND_WORD:
+                return new AddTeamCommandParser().parse(args);
+            case ListTeamCommand.COMMAND_WORD:
+                parsedIndex = ParserUtil.parseIndex(index);
+                return new ListTeamCommand(parsedIndex);
+            case DeleteTeamCommand.COMMAND_WORD:
+                parsedIndex = ParserUtil.parseIndex(index);
+                return new DeleteTeamCommand(parsedIndex);
+            case AddContactToTeamCommand.COMMAND_WORD:
+                return new AddContactToTeamCommandParser().parse(args);
```

Comment:
I think `index` should be passed to the parser? Otherwise you won't be able to get your team index.